prow/config/config_test.go

```diff
@@ -82,7 +82,7 @@ func TestContextMatches(t *testing.T) {
 
 func CheckRetest(t *testing.T, repo string, presubmits []Presubmit) {
 	for _, p := range presubmits {
-		expected := fmt.Sprintf("@k8s-bot %s test this", p.Name)
+		expected := fmt.Sprintf("/test %s", p.Name)
```

Comment:
This is good as we validate that `/test JOB` works. However we need something here that validates that `@k8s-bot JOB test this` continues to work, which we no longer have.